src/v2/guide/class-and-style.md

```diff
@@ -1,32 +1,32 @@
 ---
-title: Class and Style Bindings
+title: Enlace de Clases y Estilos
 type: guide
 order: 6
 ---
 
-A common need for data binding is manipulating an element's class list and its inline styles. Since they are both attributes, we can use `v-bind` to handle them: we just need to calculate a final string with our expressions. However, meddling with string concatenation is annoying and error-prone. For this reason, Vue provides special enhancements when `v-bind` is used with `class` and `style`. In addition to strings, the expressions can also evaluate to objects or arrays.
+Una necesidad común para atar los datos es manipulando la lista de clases de un elemento y sus estilos en línea. Dado que ambos son atributos, podemos usar `v-bind` para manipularlos: Solo necesitamos calcular una cadena final de nuestras expresiones. Sin embargo, involucrarse con concatenación de cadenas es molesto y propenso a errores. Por esta razón, Vue brinda mejoras especiales, cuando `v-bind` es usado con `class` y `style`. Además de las cadenas, las expresiones también pueden evaluar objetos o arreglos.
```

Comment:
`strings` could be better translated as `Cadena de caracteres` [source](https://es.wikipedia.org/wiki/Cadena_de_caracteres)